tests/fixtures/metadata-resp-with-embeds.json

```diff
@@ -87,6 +87,10 @@
                         "href": "http://localhost:8000/v2/providers/registrations/osf/",
                         "meta": {}
                     }
+                },
+                "data": {
+                    "id": "osf",
```

Comment:
I think this is due to the this json coming from an old OSF version, so I had to add this.